docs/GettingStartedDocs/README.md

```diff
@@ -0,0 +1,93 @@
+Getting Started With Open Enclave
+=================================
+
+Introduction
+------------
+
+Open Enclave (OE) is an SDK for building enclave applications in C and C++. An
+enclave application partitions itself into two components:
+1. An untrusted component (called the host) and
+2. A trusted component (called the enclave).
+
+An _enclave_ is a protected memory region that provides confidentiality for data
+and code execution. It is an instance of a Trusted Execution Environment (TEE)
+which is usually secured by hardware, for example,
+[Intel Software Guard Extensions (SGX)](https://software.intel.com/en-us/sgx).
+
+This SDK aims to generalize the development of enclave applications across TEEs
+from different hardware vendors. The current implementation provides support for
+Intel SGX as well as preview support for OP-TEE OS on ARM TrustZone. As an
+open source project, this SDK also strives to provide a transparent solution
+that is agnostic to specific vendors, service providers and choice of operating
+systems.
+
+Hardware Drivers
+---------------
+
+Open Enclave provides a consistent abstraction across different hardware Enclave
+platforms. If you're just getting started, you can compile and test your apps
+without an enclave (**TODO: add link to docs for this**), or you can use one of
+the hardware types below.
+
+**TODO**: start separating documentation by hardware type
```

Comment:
Not sure what this means since SGX and OP-TEE have separate docs now.  (There are certainly many issues with the docs but I don't yet follow what this TODO is.)

Reply:
I haven't read through all the docs yet, so didn't I realize that. thanks for spotting this. I'll remove this TODO.

Reply:
```suggestion
```